test/e2e_node/e2e_node_suite_test.go

```diff
@@ -91,7 +91,7 @@ type LogReporter struct{}
 func (lr *LogReporter) SpecSuiteWillBegin(config config.GinkgoConfigType, summary *types.SuiteSummary) {
 	b := &bytes.Buffer{}
 	b.WriteString("******************************************************\n")
-	glog.V(2).Infof(b.String())
+	glog.V(0).Infof(b.String())
```

Comment:
Thanks.  Should I fix this here, or just do as a follow up when working on the next piece?